lib/src/main/java/growthbook/sdk/java/DecryptionUtils.java

```diff
@@ -16,9 +19,16 @@
  * INTERNAL: This class is used internally to decrypt an encrypted features response
  */
 class DecryptionUtils {
-    public static String decrypt(String payload, String encryptionKey) {
+
+    public static class DecryptionException extends Exception {
+        public DecryptionException(String errorMessage) {
+            super(errorMessage);
+        }
+    }
+
+    public static String decrypt(String payload, String encryptionKey) throws DecryptionException {
```

Comment:
Changing the exception to `DecryptionException` so that it's checked (instead of the `IllegalArgumentException` which is a runtime error). We are handling it in the `GBContext` and `GBFeaturesRepository`.